spooky_manor2.py

```diff
@@ -61,31 +87,61 @@ def take(self, item):
     # 20: ServantsQuarters(),
     # 21: MasterSuite(),
 }
-
-print("\nWELCOME TO SPOOKY MANOR...")
+game_name = "SPOOKY MANOR"
+print(f"\nWELCOME TO:\n{game_name}...")
 show_help()
-print(Gate.intro)
 
 player_location = 1
 points = 0
 
+print(rooms[player_location].intro)
+
 while True:
-    class_name = rooms[player_location]
+    room_name = rooms[player_location]
     command = input("\n >  ").upper()
     try:
-        if type(class_name.commands[command]) == dict:
-            for key, value in class_name.commands[command].items():
-                if key == "message":
-                    print(class_name.commands.get(command)["message"])
+        result = room_name.commands
```

Comment:
This isn't the result. You're storing the list of commands in result. Instead, it should be:
```python
result = room_name.commands.get(command)
```